docs/deployment/README.md

```diff
@@ -320,7 +313,7 @@ A successful completion will result in output similar to:
   $ DEPLOYMENT=<cumulus-deployment-name> \
       AWS_REGION=<region> \ # e.g. us-east-1
       AWS_PROFILE=<profile> \
-      npm run deploy
+      npm run deploy-app
```

Comment:
Out of curiosity, does the deployment output still resemble what we have below? If not maybe we should update it

Reply:
No, but it does get very long with all the sub-templates in our example. I'm not sure it's ideal to put all that in a doc. I can try generating some off the template-deploy.